boards/BMS/Inc/Io/Io_LTC6813.h

```diff
@@ -0,0 +1,36 @@
+#pragma once
+
+#include <stm32f3xx_hal.h>
+#include "App_SharedExitCode.h"
+
+/**
+ * Initialize the LTC6813 daisy chain.
+ * @param hspi The SPI handle for the LTC6813 daisy chain.
+ * @param nss_port A pointer to the GPIO port corresponding to the LTC6813 daisy
+ * chain.
+ * @param nss_pin The GPIO pin corresponding to the LTC6813 daisy chain.
+ */
+void Io_LTC6813_Init(
+    SPI_HandleTypeDef *hspi,
+    GPIO_TypeDef *     nss_port,
+    uint16_t           nss_pin);
+
+/**
+ * Configure all LTC6813 chips on the daisy chain.
+ */
+void Io_LTC6813_Configure(void);
+
+/**
+ * Read all cell register groups for each chip on the daisy chain.
+ * @return EXIT_CODE_OK if no PEC15 mismatches occur. Else, return
+ * EXIT_CODE_ERROR.
+ */
+ExitCode Io_LTC6813_ReadAllCellRegisterGroups(void);
+
+/**
+ * Get the pointer to the double array containing converted cell voltages for
+ * each chip on the daisy chain.
+ * @return The pointer to the double array containing converted chip cell
+ * voltages.
```

Comment:
How long is this array? Is the caller responsible for de-allocating it?

Reply:
We probably need to return a number of cells as well if it can change. Otherwise we should specify a constant.

Reply:
This should be referring to the `cell_voltages` 2D array in the LTC6813 struct.

Reply:
When you say specify a constant do you mean something like [this](https://github.com/UBCFormulaElectric/Consolidated-Firmware/pull/672/files/b757ab004a8f6b3a243ff1a06a49d04c5c5e09cd#r497274956) in the javadoc?

Reply:
@garethellis0 There's a constant that I used to define the size of the array. Did you want me to add the constant to the javadoc?

Reply:
I added it to the javadoc anyway since I think this would be good information for a user glancing through the .h file